src/StudyUtils.in.jsm

```diff
@@ -335,7 +335,7 @@ class StudyUtils {
     return queryArgs;
   }
 
-  _telemetry(data, bucket = "shield-study-addon") {
+  async _telemetry(data, bucket = "shield-study-addon") {
```

Comment:
This is fine.  Wasn't sure if it had a useful return value or not!  EVERYTHING with Telemetry ID is async, fwiw.